src/Components/App/App.js

```diff
@@ -0,0 +1,107 @@
+import { Component } from 'react';
+import { getRandomFiveLetterWord, findWordInAPIDatabase } from '../../apiCalls';
+import Header from '../Header/Header';
+import GameBoardContainer from '../GameBoardContainer/GameBoardContainer';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      currentWordInPlay: null,
+      typedLetters: [],
+      submittedWords: [],
+      gameOver: false,
+      playerStats: [],
+      error: null
+    }
+  }
+
+  componentDidMount = () => {
+    return getRandomFiveLetterWord()
+      .then(data => this.setState({ currentWordInPlay: data.word }))
+      .catch(error => {
+        this.setState({ error: error.message })});
+  }
+
+  typeLetter = (letter) => {
+    if (this.state.typedLetters.length < 5 && !this.state.gameOver) {
+      const updatedTypedLetters = [...this.state.typedLetters, letter];
+      this.setState({ typedLetters: updatedTypedLetters });
+    }
+  }
+
+  deleteLetter = () => {
+    if (!this.state.gameOver) {
+      const updatedTypedLetters = [...this.state.typedLetters];
+      updatedTypedLetters.pop();
+      this.setState({ typedLetters: updatedTypedLetters, error: null });
+    }
+  }
+
+  validateGuess = (word) => {
+    return findWordInAPIDatabase(word)
+      .then(response => response.status === 200)
+      .catch(error => this.setState({ error: error.message }));
+  
+  }
+  
+  enterGuess = () => {
+    const guess = this.state.typedLetters.join('');
```

Comment:
I was thinking about your project last night and was thinking about the fact that you were probably going to get lots of practice with mutator methods!  It already seems that way ☺️

Reply:
Yes, building out the actual game felt a lot like doing CodeWars.